lib/MusicBrainz/Server/Edit/Event/Edit.pm

```diff
@@ -110,6 +112,16 @@ sub build_display_data
         }
     }
 
+    if (exists $data->{ended}) {
+        $data->{ended}{old} = boolean_to_json($data->{ended}{old});
+        $data->{ended}{new} = boolean_to_json($data->{ended}{new});
+    }
```

Comment:
I wouldn't bother with this, since we're not displaying it, unless we have to for some reason.